src/interfaces/permissions/request/method.ts

```diff
@@ -0,0 +1,40 @@
+import type { InterfaceMethod } from '../../types';
+import type { JsonPermissionsRequest } from './types';
+
+import { MessageResult } from '../../../response';
+import { PermissionsRequest } from './message';
+
+export const processPermissionsRequest: InterfaceMethod = async (
```

Comment:
Nit: if you took the rename `method.ts` -> `handler.ts`, then would be nice to be consistent with terminology:

```suggestion
export const handlePermissionsRequest: InterfaceMethod = async (
```

Reply:
done